lib/unleash/client.rb

```diff
@@ -59,24 +59,26 @@ def if_enabled(feature, context = nil, default_value = false, &blk)
     def get_variant(feature, context = nil, fallback_variant = nil)
       Unleash.logger.debug "Unleash::Client.get_variant for feature: #{feature} with context #{context}"
 
+      default_variant = fallback_variant || Unleash::FeatureToggle.new.disabled_variant
+
       if Unleash.configuration.disable_client
         Unleash.logger.debug "unleash_client is disabled! Always returning #{default_variant} for feature #{feature}!"
```

Comment:
I'm thinking that it would be a better approach would be just to just assign `Unleash::FeatureToggle.disabled_variant` in the method signature as the default value instead of nil.

Note that you don't want to instantiate a new object, to get a hold of the disabled variant. It is possible if the disabled_variant method is static. (see comment below in `FeatureToggle.rb`).

Reply:
You will also need to rename referenced variable in the log message from `default_variant` to `fallback_variant`.